resources/views/site/home.blade.php

```diff
@@ -22,7 +22,27 @@
 </div>
 <div class="container-fluid bg-secondary">
 	<div class="row">
-		<div class="col-sm-9">
+        <div class="col-lg-2 pr-md-2 pb-3 pb-lg-0">
```

Comment:
Can you flex this col so it matches the length of the carousel? It looks odd when there are no event scheduled (the banner carousel is much taller in the viewport)